src/ussdapp.js

```diff
@@ -211,45 +217,25 @@ go.app = function() {
 
         // 004
         self.add('state_who_refugee', function(name) {
-            return new ChoiceState(name, {
-                question: $('CONTENT 004'),
-                choices: [
-                    new Choice('refugee', $('Yes, I am a refugee')),
-                    new Choice('back', $('No, back to menu'))
-                ],
-                next: function(choice) {
-                    if (choice.value === 'back') {
-                        return 'state_status';
-                    } else {
-                        return go.utils
-                            .register_user(self.contact, self.im, choice.value)
-                            .then(function() {
-                                return 'state_refugee_rights_info';
-                            });
-                    }
-                }
+            return new PaginatedState(name, {
+                text: $("If you fled from your country in fear of your life due to your race, religion, nationality, gender, political or social group. Or; if your life, safety or freedom in your home country are at risk because of violence, war & civil unrest. Or; if you are married to or depend upon a person who fled their country in fear of their life for the reasons listed. You are entitled to refugee status if you are married to a recognised refugee, even if your own claim was rejected."),
+                characters_per_page: 160,
+                back: $('<-'),
```

Comment:
No. I added it here as a UI experiment to be evaluated by the Praekelt team on USSD.  I did it here since the default 'Exit' doesn't make intuitive sense here for going back to the previous screen.